Mathlib/Logic/Equiv/Fin/Rotate.lean

```diff
@@ -108,3 +110,11 @@ theorem finRotate_symm_lt_iff_ne_zero [NeZero n] (i : Fin n) :
   · simp only [hc, Fin.not_lt_zero] at hi
   · rw [Fin.lt_iff_val_lt_val, coe_finRotate_symm_of_ne_zero hi]
     apply sub_lt (zero_lt_of_ne_zero <| Fin.val_ne_zero_iff.mpr hi) Nat.zero_lt_one
+
+/-- The permutation on `Fin n` that adds `k` to each number. -/
+@[simps]
+def finCycle (k : Fin n) : Equiv.Perm (Fin n) where
```

Comment:
Is this the same as `finRotate n ^ i.val`?

Reply:
Yes, but I do not need this equivalence for the proof.